src/pages/Home.js

```diff
@@ -1,9 +1,34 @@
 import React, { Component } from 'react';
+import * as Api from '../services/api';
+import CategoryList from '../components/categoryList';
 
 class Home extends Component {
+  constructor() {
+    super();
+
+    this.state = {
+      categories: [],
+      loading: true,
+    };
+  }
+
+  componentDidMount() {
+    this.requiredCategories();
+  }
+
+  async requiredCategories() {
+    const request = await Api.getCategories();
+    this.setState({
+      categories: request,
+      loading: false,
+    });
+  }
```

Comment:
Que tal renomear essa funcao `requiredCategories` para `fetchCategories` para melhorar o significado? 🙂
```suggestion
  componentDidMount() {
    this.fetchCategories();
  }

  async fetchCategories() {
    const request = await Api.getCategories();
    this.setState({
      categories: request,
      loading: false,
    });
  }
```